BE/PoroDev.Common/Exceptions/IdFormatexception.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PoroDev.Common.Exceptions
+{
+    public class IdFormatexception : Exception
+    {
+        public string HumanReadableErrorMessage { get; set; }
+
+        public IdFormatexception() : base()
+        {
+            HumanReadableErrorMessage = "Internal server error occurred";
```

Comment:
Could the message be more specific here?

Reply:
We always call the constructor with the parameter which has a human-readable message, so this constructor is there only if an exception is thrown somewhere where we didn't predict it, which basically is an internal server error. So I think it is okay to leave it like this